### PR TITLE
Stress tests ble

### PR DESCRIPTION
**Pull request by mkasenberg**
5 tests done. 3 more to go.

### CODE REVIEW

**Comment by mkasenberg** [comment]
@kolodgrz @MariuszSkamra @szymon-czapracki

File: apps/blestress/src/rx_stress.c [hunk 1]
```diff
@@ -0,0 +1,876 @@
...
+        .pattern_len = 0,
+    },
+};
+
+static void
+rx_stress_9_perform()
+{
+    do {
+        /* Test is based on gap event handling, so start advert. */
+        rx_stress_simple_adv(rx_stress_adv_sets + 9);
```
Comment (kolodgrz) [inline apps/blestress/src/rx_stress.c]: 9 is for what ?

File: apps/blestress/src/rx_stress.c [hunk 2]
```diff
@@ -0,0 +1,876 @@
...
+        break;
+    case 8:
+        rx_stress_simple_adv(rx_stress_adv_sets + 8);
+        break;
+    case 9:
+        rx_stress_9_perform();
+        break;
+    default:
+        console_printf("\033[0;31mFound test, but do not know how to perform."
+                       "\033[0m\n");
```
Comment (kolodgrz) [inline apps/blestress/src/rx_stress.c]: maybe assert should be here

File: apps/blestress/src/rx_stress.c [hunk 3]
```diff
@@ -0,0 +1,876 @@
...
+        /* Start test. */
+        rx_stress_start(i);
+    }
+
+    /* Print tests results */
+    com_stress_print_report(rx_stress_ctxs);
+
+    /* Task should never return */
+    while (1) {
+        os_time_delay(os_time_ms_to_ticks32(OS_TIME_MAX));
```
Comment (kolodgrz) [inline apps/blestress/src/rx_stress.c]: why this delay is needed here ? isn't while (1) enough ? - what is asm difference ? - how looks while(1) ?

File: apps/blestress/src/rx_stress.c [hunk 4]
```diff
@@ -0,0 +1,876 @@
...
+    }
+}
+
+void
+rx_stress_start_auto()
+{
+    /* Start task that will run all stress tests one by one. */
+    os_task_init(&rx_stress_main_task, "rx_stress_main_task",
+                 rx_stress_main_task_fn, NULL, 0xf0, OS_WAIT_FOREVER,
```
Comment (kolodgrz) [inline apps/blestress/src/rx_stress.c]: why prio should be 0xf0 ?

**Comment by rymanluk** [comment]
Thanks @mkasenberg

**Pull request merged**